openapi-v2.yaml

```diff
@@ -2626,6 +2699,64 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /groups/{namespace}/{group}/activity:
+    parameters:
+      - name: namespace
+        in: path
+        description: namespace group is in (an organization name or user's username)
+        type: string
+        required: true
+      - name: group
+        in: path
+        description: name/uri of group that is url-encoded
+        type: string
+        required: true
+      - name: start
+        in: query
+        description: "Start time of window of fetch logs, unix epoch in seconds (default: seven days ago)"
+        type: integer
+        format: uint64
+        required: false
+      - name: end
+        in: query
+        description: "End time of window of fetch logs, unix epoch in seconds (default: current utc timestamp)"
+        type: integer
+        format: uint64
+        required: false
+      - name: event_types
+        in: query
+        collectionFormat: multi
+        type: array
+        items:
+          type: string
+        description: Refer to GroupActivityEventType for possible values
+      - name: page
+        in: query
+        description: pagination offset
+        type: integer
+        required: false
+      - name: per_page
+        in: query
+        description: pagination limit
+        type: integer
+        required: false
+    get:
+      description: get group activity logs
+      tags:
+        - group
+      operationId: groupActivityLog
```

Comment:
I would name this listGroupActivity or something

Reply:
Fixed